.storybook/main.js

```diff
@@ -36,6 +39,17 @@ module.exports = {
 			],
 		});
 		config.resolve.plugins ||= [];
+		config.plugins.push(
+			// We rely on Buffer for our bridget thrift client
```

Comment:
Just for my own understanding what is a bridget thrift client?

Reply:
Sorry, this was me accidentally copying WebX's comment along with the implementation! I've added a more comprehensive comment for the MMA usecase.